goshopify.go

```diff
@@ -344,7 +351,7 @@ func (c *Client) doGetHeaders(req *http.Request, v interface{}) (http.Header, er
 
 func wrapSpecificError(r *http.Response, err ResponseError) error {
 	if err.Status == 429 {
-		f, _ := strconv.ParseFloat(r.Header.Get("retry-after"), 64)
+		f, _ := strconv.ParseFloat(r.Header.Get("Retry-After"), 64)
```

Comment:
http headers are case insensitive - this shouldn't make a difference.

Reply:
a previous merge brought in uppercase, i was just updating this one to be consistent.

Reply:
👍 - makes sense.